### ISSUE TITLE
Jest 28 Support

### ISSUE THREAD

**Issue by incetarik** [relmify/jest-fp-ts#7]
**Describe the issue**
Jest version 28 has been released and provides several improvements. Yet, because of the dependencies removed from jest, this library becomes incompatible with it.

**To Reproduce**
Simply upgrade your Jest to version 28.

**Expected behavior**
The library would be compatible with Jest 28.

**Suggested fix**
Removing unused dependencies by Jest 28.

**Additional context**
The error returned is similar to this:

```
Cannot find module 'expect/build/jasmineUtils' from '@relmify/jest-fp-ts/dist/util/index.js
```

**Comment by leilapearson** [comment]
Thanks for the report. It's actually a good thing that they made this change since they now explicitly provide a package with the functions I need. I should have an update out shortly.

**Comment by leilapearson** [comment]
Fixed in release 2.0.2
